test/extrema_test.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// extrema_test.dart
+// Tests for extrema.
+//
+// 2024 September 16
+// Author: Roberto Torres <roberto.torres@intel.com>
+//
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+
+void main() {
+  tearDown(() async {
+    await Simulator.reset();
+  });
```

Comment:
some additional cases to consider:
- if the max/min is in there multiple times, does it correctly find the first one?
- is it able to correctly find the max/min when it is the first/last element in the list?